Products/CMFPlone/tests/redirection.txt

```diff
@@ -31,5 +31,16 @@ But it should not give an error while rendering the default error page:
 
   >>> "the following error occurred while attempting to render the standard error message" in browser.contents
   False
+
+As it is am image, it should return a JSON message
+  >>> '{"error_type": "NotFound"}' in browser.contents
```

Comment:
well, at first I agreed, but:

- apparently this information is not available in `Products/CMFPlone/skins/plone_templates/standard_error_message.py`
- the HTTP status is already prviding this information, there is no need to repeat here, so I thought the error type might be interesting